components/Footer.vue

```diff
@@ -76,8 +76,137 @@
+        <ul class="sp-footer-item" :class="{ '_state-open': isOpened[1] }" v-if="isOpened[1]">
+          <li>
+            <a href="#">AIESECについて</a>
+          </li>
+          <li>
+            <a href="#">パートナーシップについて</a>
+          </li>
+          <li>
+            <a href="#">団体情報</a>
+          </li>
+          <li>
+            <a href="#">安全への取り組み</a>
+          </li>
+          <li>
+            <a href="#">個人情報の取り扱い</a>
+          </li>
+          <li>
+            <a href="#">お問い合わせ</a>
+          </li>
+        </ul>
+      </transition>
+    </div>
+    <div class="sp-footer-content">
+      <button type="button" class="sp-footer-title" :class="{ '_state-open': isOpened[2] }" @click="openItems(2)">
+        For Students
+      </button>
+      <transition name="sp-footer-item" @before-enter="beforeEnter" @enter="enter" @before-leave="beforeLeave" @leave="leave">
+        <ul class="sp-footer-item" :class="{ '_state-open': isOpened[2] }" v-if="isOpened[2]">
+          <li>
+            <a href="#">海外インターンシップに参加する</a>
+          </li>
+          <li>
+            <a href="#">海外ボランティアに参加する</a>
+          </li>
+          <li>
+            <a href="#">体験記を見る</a>
+          </li>
+          <li>
+            <a href="#">参加の流れと費用</a>
+          </li>
+          <li>
+            <a href="#">危機管理サポート</a>
+          </li>
+          <li>
+            <a href="#">話を聞いてみる</a>
+          </li>
+        </ul>
+      </transition>
+    </div>
+    <div class="sp-footer-content">
+      <button type="button" class="sp-footer-title" :class="{ '_state-open': isOpened[3] }" @click="openItems(3)">
+        For Companies
+      </button>
+      <transition name="sp-footer-item" @before-enter="beforeEnter" @enter="enter" @before-leave="beforeLeave" @leave="leave">
+        <ul class="sp-footer-item" :class="{ '_state-open': isOpened[3] }" v-if="isOpened[3]">
+          <li>
+            <a href="#">海外インターン生を受け入れる</a>
+          </li>
+          <li>
+            <a href="#">受け入れ事例を見る</a>
+          </li>
+          <li>
+            <a href="#">受け入れの流れと費用</a>
+          </li>
+        </ul>
+      </transition>
+    </div>
+    <div class="sp-footer-content">
+      <button type="button" class="sp-footer-title" :class="{ '_state-open': isOpened[4] }" @click="openItems(4)">
+        Press & Alumni
+      </button>
+      <transition name="sp-footer-item" @before-enter="beforeEnter" @enter="enter" @before-leave="beforeLeave" @leave="leave">
+        <ul class="sp-footer-item" :class="{ '_state-open': isOpened[4] }" v-if="isOpened[4]">
+          <li>
+            <a href="#">プレスルーム</a>
+          </li>
+          <li>
+            <a href="#">アルムナイの方</a>
+          </li>
+        </ul>
+      </transition>
+    </div>
+  </div>
 </template>
 
+<script>
+export default {
+  props: [
+    "width"
+  ],
+  data() {
+    return {
+      isOpened: [false,false,false,false,false]
+    };
+  },
+  methods: {
+    openItems: function(index) {
```

Comment:
`: functions` は省略できるのでこっちで書きましょう！

```suggestion
    openItems(index) {
```